tests/appsec/iast/sink/test_weak_cipher.py

```diff
@@ -10,24 +10,9 @@
 
 
 @coverage.basic
-@released(dotnet="?", golang="?", php_appsec="?", python="1.7.0", ruby="?")
+@released(dotnet="?", golang="?", java="0.108.0", php_appsec="?", python="1.7.0", ruby="?")
```

Comment:
Why are removed ?

"resteasy-netty3": "1.11.0",
"jersey-grizzly2": "1.11.0",
"vertx3": "1.12.0",
"akka-http": "1.12.0"

Reply:
Yep. Confirmed. For example, vertx4 works in dd-trace-java 1.7.0, it just depends on the weblog having the endpoint implemented, but the vuln is fine with older tracers.

Reply:
ok!! thanks!